### DISCUSSION TITLE
Question about sub_system_coupling

### DISCUSSION THREAD

**Discussion by theodoho** [dschick/udkm1Dsim#143] [Q&A] [answered]
I have a question about the _sub_system_coupling_ parameter:

Is this parameter the same as Gj(t) in the equation solved by _calc_heat_diffusion_ (https://udkm1dsim.readthedocs.io/en/latest/heat.html#udkm1Dsim.simulations.heat.Heat.calc_heat_diffusion)?

This is not stated in the documentation, so I just want to get a confirmation on this.

**Comment by dschick** [comment, marked as answer]
thanks @theodoho for asking that question.
Yes you are right, the `sub_system_coupling` term is the same as the `G_j(T)` in the heat diffusion equation.
I will add this to the according docstrings.

Note that the argument `T` of the  `sub_system_coupling` is treated as a vector so you can access the individual temperatures `T_i` of each of the other substystems.

This is currently not possible for all other thermoelastic parameters, such as the `heat_capacity, `thermal_conductivity`, or `linear_thermal_expansion`. They have only access to the individual temperature of the current sub-system.
This limitation will be also tackled in the future, see #70.

**Comment by dschick** [comment]
solved by https://github.com/dschick/udkm1Dsim/commit/b12a80e2db2eac5b4277839759c3655babf99b1e
